src/pxl_scripts/bpftrace/socket_size/data.pxl

```diff
@@ -0,0 +1,177 @@
+# Adapted From "BPF Performance Tools" book by Brendan Gregg; Chapter 10: Networking
```

Comment:
The pxl script as a whole also needs the Pixie author license added. 
Can you please follow the other scripts as a guide or the linter output and add it.